### ISSUE TITLE
Bad error message when rows in VALUES have different number of fields

### ISSUE THREAD

**Issue by martint** [prestodb/presto#5342]
```sql
VALUES (1), (1,2)
```

fails with:

```
java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
	at java.util.ArrayList.get(ArrayList.java:429)
	at com.facebook.presto.sql.analyzer.StatementAnalyzer.visitValues(StatementAnalyzer.java:1429)
	at com.facebook.presto.sql.analyzer.StatementAnalyzer.visitValues(StatementAnalyzer.java:229)
	at com.facebook.presto.sql.tree.Values.accept(Values.java:55)
	at com.facebook.presto.sql.tree.AstVisitor.process(AstVisitor.java:22)
	at com.facebook.presto.sql.analyzer.StatementAnalyzer.visitQuery(StatementAnalyzer.java:928)
	at com.facebook.presto.sql.analyzer.StatementAnalyzer.visitQuery(StatementAnalyzer.java:229)
	at com.facebook.presto.sql.tree.Query.accept(Query.java:103)
	at com.facebook.presto.sql.tree.AstVisitor.process(AstVisitor.java:22)
	at com.facebook.presto.sql.analyzer.Analyzer.analyze(Analyzer.java:60)
	at com.facebook.presto.execution.SqlQueryExecution.doAnalyzeQuery(SqlQueryExecution.java:273)
	at com.facebook.presto.execution.SqlQueryExecution.analyzeQuery(SqlQueryExecution.java:259)
	at com.facebook.presto.execution.SqlQueryExecution.start(SqlQueryExecution.java:223)
	at com.facebook.presto.execution.QueuedExecution.lambda$start$1(QueuedExecution.java:62)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by javainthinking** [comment]
Hi @martint , I changed some code for this issue at pull request: https://github.com/prestodb/presto/pull/5409
Does it make sense?

**Comment by philippu** [comment]
Fixed in 7e1336b. Closing this.
